recipe/run_test.py

```diff
@@ -8,7 +8,7 @@
 import scipy
 import scipy.cluster._hierarchy
 import scipy.cluster._vq
-import scipy.fftpack._fftpack
+# import scipy.fftpack._fftpack  # this seems to not be there as of 1.4.0
```

Comment:
This line should just be removed, IMO. Maybe add a test for importing the replacement module `scipy.fft` see [release notes](https://github.com/scipy/scipy/releases), first bullet point under highlights.

Reply:
Sounds good.